ably/ably_test.go

```diff
@@ -261,6 +261,24 @@ func (rec *MessageRecorder) CheckIfSent(action ably.ProtoAction, times int) func
 	}
 }
 
+func (rec *MessageRecorder) CheckIfReceived(action ably.ProtoAction, times int) func() bool {
+	return func() bool {
+		counter := 0
+		for _, m := range rec.Received() {
+			if m.Action == action {
+				counter++
+				if counter == times {
+					return true
+				}
+			}
+		}
+		if times == 0 && times == counter {
```

Comment:
what does this do?

Reply:
If 0 is passed as times integer, that means it expects given proto msg type to be called 0 times.

Reply:
Also means, we are not expecting protoMsg with given action to be received.

Reply:
would it be clearer as 
`if times == 0 && counter == 0 {`
?

Reply:
updated https://github.com/ably/ably-go/pull/661/commits/bd39bc370450449fd2f6140d721428ca69173b5b